### DISCUSSION TITLE
Version 2.0

### DISCUSSION THREAD

**Discussion by CodeAsherPlayz** [PythonMathCalculator/python-math-calculator#14] [Publishing]
@rutvikpadhiyar000 is this version number okay? I think we can publish this first in the Release panel right?

**Comment by CodeAsherPlayz** [comment]
@rutvikpadhiyar000 can we publish this Version?

**Reply by ghost** [reply]
I think so

**Reply by CodeAsherPlayz** [reply]
Alright, thanks. But I need you to help me with something. If you do not know, I'm a teenager and school in Singapore starts on Monday. I would like to ask you to be a Maintainer in this repository. @rutvikpadhiyar000

**Reply by ghost** [reply]
Same here bro my college starts on Monday too

**Reply by ghost** [reply]
I think we can each take some time and do it togather

**Reply by CodeAsherPlayz** [reply]
Yeah but I am quite busy when school starts.

**Reply by CodeAsherPlayz** [reply]
I will peek in once in a while or when i have time. I have the GitHub app on my phone but it doesn't have all the features yet.

**Reply by CodeAsherPlayz** [reply]
> Same here bro my college starts on Monday too

Ahh haha so it is too young people coding

**Reply by CodeAsherPlayz** [reply]
Okayokay maybe I make a schedule for us

**Reply by CodeAsherPlayz** [reply]
I'll see first

**Reply by ghost** [reply]
Ok

**Comment by ghost** [comment]
Hey can you archive version 1.0
Syntex is something like: http://stackoverflow.com/questions/1307114/ddg#1309934

**Reply by CodeAsherPlayz** [reply]
As in the branch or the release?

**Reply by ghost** [reply]
I made branch version 1.0
Just archive that and remove version 1 from current release

**Reply by CodeAsherPlayz** [reply]
Okay but we don't need to archive version 1.0

**Reply by CodeAsherPlayz** [reply]
Just let it stay there in case people want to download older versions. (as in releases)

**Reply by ghost** [reply]
That way we save all version and still everyone will know which version is currently maintained

**Comment by ghost** [comment]
Archiving it will make it read only so anyone can download older version but cant create issue or pull request.

**Reply by CodeAsherPlayz** [reply]
I see...

**Comment by ghost** [comment]
I will do it later and than you review it

**Reply by CodeAsherPlayz** [reply]
Okay I actually gave you maintain permission but we should make a pull request

**Reply by ghost** [reply]
Ya sorry for that branch i actually didn't notice where i was working.

**Reply by CodeAsherPlayz** [reply]
It's okay :D

**Comment by CodeAsherPlayz** [comment]
@rutvikpadhiyar000 so can you actually archive Version 1.0?

**Comment by ghost** [comment]
It seems so let me try it but i think i will have to work on reposetory.

**Comment by ghost** [comment]
It seems so but if you go to tags there are already saves versions of repository of the time when new version is published so we can delete older versions from our current repository

**Reply by CodeAsherPlayz** [reply]
Uhhhhh... I think we should leave it as it is and delete the archives.

**Reply by CodeAsherPlayz** [reply]
I can fix the older versions as I have a backup version locally on my desktop

**Reply by ghost** [reply]
Yeah thats good.
